### PR TITLE
add bespin_lando_worker from gcb-ansible

### PR DESCRIPTION
**Pull request by johnbradley**
Copied this role from the [gcb-ansible](https://github.com/Duke-GCB/gcb-ansible) repo.
Includes modifications to be more flexible.
Variable `lando_version` specifies the version of lando to install.
Variable `cwl_runner_version` specifies the version of cwltool to install( or whatever tool is specified via `cwl_runner)`.

### CODE REVIEW

File: bespin_lando_worker/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+lando_worker:
+  lando_version: 0.9.3
+  cwltool_version: 1.0.20180912090223
+  work_directory: /work
+  path_to_lando_worker: /usr/local/bin/lando_worker
```
Comment (dleehr) [inline bespin_lando_worker/defaults/main.yml]: While I agree the path to the lando worker command is "magic", I'm not sure there's a use case for exposing it to be changed.

I believe the location of this file depends on where `pip3` puts it when it installs lando. Perhaps the better way to deal with the magic is to document that in the role, and maybe register the output of `which lando_worker` to get the absolute path for injecting into the startup script?

**Comment by dleehr** [comment]
Looks like I commented on an earlier iteration, but the point still stands (even though Github thinks its outdated)

**Comment by johnbradley** [comment]
That's weird the file hasn't changed.

**Comment by dleehr** [comment]
> That's weird the file hasn't changed.

Well I did comment on the older commit. The flattening in ee9fe92be397959898319411a8deba20335ad2da makes it outdated. I was just looking at where that was added. So GitHub didn't do anything weird here, @dleehr did.

**Pull request merged**